yarn-project/prover-client/src/config.ts

```diff
@@ -29,7 +29,7 @@ export function getProverEnvVars(): ProverConfig {
     ACVM_BINARY_PATH = '',
     BB_WORKING_DIRECTORY = tmpdir(),
     BB_BINARY_PATH = '',
-    PROVER_AGENTS = '1',
+    PROVER_AGENTS = '10',
```

Comment:
I'm not sure we should set this default here. I'm always mindful that defaults should be a value that would make sense in a production setting. Can we not just set this for test runs?